scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -1044,8 +1070,14 @@ def _benchmark_cnn(self):
       # in replicated mode).
       ready_for_local_init_op = tf.report_uninitialized_variables(
           tf.global_variables())
+    if self.params.variable_update == 'horovod':
+      import horovod.tensorflow as hvd
+      bcast_global_variables_op = hvd.broadcast_global_variables(0)
+    else:
+      bcast_global_variables_op = None
     sv = tf.train.Supervisor(
-        is_chief=is_chief,
+        # For the purpose of Supervisor, all Horovod workers are 'chiefs'.
```

Comment:
is there more that can be said here (or maybe there is a link that would explain why this is the case?)?